.ci_support/linux_64_grpc_cpp1.47.yaml

```diff
@@ -13,7 +13,7 @@ cxx_compiler_version:
 docker_image:
 - quay.io/condaforge/linux-anvil-cos7-x86_64
 grpc_cpp:
-- '1.48'
+- '1.47'
```

Comment:
Hmm, weird. Something might've gone off with  the re-render here?

Reply:
@mariusvniekerk pointed out the relevant migration was rolled back so this is expected